cactus.rb

```diff
@@ -0,0 +1,177 @@
+class Cactus
+  attr_reader :ascii
+
```

Comment:
I really like that this lives in its own file. This helps keep your code organized, and means the reader doesn't have to wade through a hundred lines of ASCII art before getting to the meat of your program.

In general separating logic and data is a very useful technique, and one we'll continue to explore throughout the course.